pkg/query-service/app/inframetrics/hosts.go

```diff
@@ -21,23 +19,45 @@ type HostsRepo struct {
 	querierV2 interfaces.Querier
 }
 
-var pointAttrsToIgnore = []string{
-	"state",
-	"cpu",
-	"device",
-	"direction",
-	"mode",
-	"mountpoint",
-	"type",
-	"process.cgroup",
-	"process.command",
-	"process.command_line",
-	"process.executable.name",
-	"process.executable.path",
-	"process.owner",
-	"process.parent_pid",
-	"process.pid",
-}
+var (
+	// we don't have a way to get the resource attributes from the current time series table
+	// but we only want to suggest resource attributes for system metrics,
+	// this is a list of attributes that we skip from all labels as they are data point attributes
+	// TODO(srikanthccv): remove this once we have a way to get resource attributes
+
+	pointAttrsToIgnore = []string{
+		"state",
+		"cpu",
+		"device",
+		"direction",
+		"mode",
+		"mountpoint",
+		"type",
+		"os_type",
+		"process_cgroup",
+		"process_command",
+		"process_command_line",
+		"process_executable_name",
+		"process_executable_path",
+		"process_owner",
+		"process_parent_pid",
+		"process_pid",
```

Comment:
do you want to keep these as constants to be reused somewhere else (if there's a need)

Reply:
I am going to remove them soon (when we have a way to easily distinguish metric data point attributes and resource attributes) so I don't see the reusability of aspect of it.